### ISSUE TITLE
About project update plan

### ISSUE THREAD

**Issue by David9591** [abtExp/arxivr#4]
Dear Author: 
Will you continue to update this project? I found the SC-FEGAN is interesting, and I thought it might be more convenient and efficient to use Pytorch to build this project.

Best wishes

**Comment by abtExp** [comment]
hi @David9591 , thanks for your interest in this project. I do plan to continue this project, currently I'm working on creating a framework [(ProjectAIle)](https://github.com/explabs-ai/projectaile) for removing the factor of writing boilerplate stuff again and again so that I can smooth out the process of reproducing a paper, I plan to provide implementations in both TensorFlow(keras) and Pytorch, but it's been a rough year and I've lost track of this and several other projects, i plan to be back on track soon, hopefully by the next month. I'll update the SC-FEGAN first and then other existing papers, and then i plan to add one paper a week that i pick from either a poll on twitter or from arxiv-sanity. I hope you find it useful.

**Comment by manjusaka999** [comment]
> Dear Author:
> Will you continue to update this project? I found the SC-FEGAN is interesting, and I thought it might be more convenient and efficient to use Pytorch to build this project.
> 
> Best wishes

The author said he would update the project this week.I think he's doing the final tests,I hope he finishes this work successfully.
Best wishes! @abtExp

**Comment by abtExp** [comment]
hi @David9591 @manjusaka999 , i just pushed the implementation to the repo, do check it out, however there are a few tensorflow bugs remaining that i need to sort out to start training and add the trained weights, i'll be doing that as soon as i get time. in the meantime do check the code, i've finished the model code, the training code and the loss functions and all the implementations are accurate to the best of my understanding. I'll be patching the fixes along with the trained weights of the model and pytorch implementation ASAP.

**Comment by David9591** [comment]
Dear author, Hi! @abtExp ,I have downloaded the code and made some attempts. My system environment is Tensorflow-gpu  2.0.0+Keras 2.3.1+cuda 10.0，For the time being, I adjusted some grammar problems in different versions. But I still had a lot of clueless problems, I think this may be due to a variety of reasons. Can you share your environment configuration? I will create a new virtual environment to investigate.

**Comment by StrongandSurvive** [comment]
Hi,men. Will you continue to update this project？As a novice, I need a detailed Readme to help me quickly learn about this project. May I follow the update plan?Best wishes!  @abtExp
